client/session_fetch_bulk_blocks_test.go

```diff
@@ -589,7 +589,7 @@ func TestSelectBlocksForSeriesFromPeerBlocksMetadataAllPeersSucceed(t *testing.T
 	session := s.(*session)
 
 	var (
-		metrics          = session.streamFromPeersMetricsForShard(0, resultTypeTest)
+		metrics          = session.streamFromPeersMetricsForShard(0, resultTypeRaw)
```

Comment:
Yep.`resultTypeTest` was only being used by tests, changed it to `resultTypeRaw` and eliminated `resultTypeTest`